modules/storage.js

```diff
@@ -0,0 +1,27 @@
+export default class Storage {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Storage.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(book) {
+    const bookTitle = book.querySelector('#title').innerText;
+    const books = Storage.getBooks();
+    const filterBooks = books.filter((book) => bookTitle === book.title);
+    const filterIndex = books.indexOf(filterBooks[0]);
+    books.splice(filterIndex, 1);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
```

Comment:
- Good job for using normal function here +1 . But for best practice following the ES6 syntaxes, you should use arrow syntax for your functions. Your functions should look like this: clearFields = () => {...}. Kindly address that. grin